src/Core/src/Core/IPicker.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Maui
+{
+	public interface IPicker : IView
+	{
+		string Title { get; }
+		IList<string> Items { get; }
+		IList ItemsSource { get; }
```

Comment:
Do these two concepts mean anything at the core layer? This feels a bit binding-y.  Also, how will Comet do this? Are we maybe forcing something?